agent-install/agent-install.sh

```diff
@@ -637,6 +638,17 @@ function install_linux(){
         log_info "jq installed"
 	fi
 
+    log_info "Checking if docker is installed..."
+    if command -v docker >/dev/null 2>&1; then
+        log_info "docker found"
+    else
+        log_info "docker not found, installing it..."
+        set -x
+        apt-get install -y docker-ce docker-ce-cli containerd.io
```

Comment:
The jq install uses apt command, here it uses apt-get command. Can it be consistent?

Reply:
Done!